doc/modules/linear_model.rst

```diff
@@ -665,6 +665,8 @@ hyperparameters :math:`\lambda_1` and :math:`\lambda_2`.
    :align: center
    :scale: 50%
 
+ARD is also known in the literature as *Sparse Bayesian Learning* and
+*Relevance Vector Machine* [3]_ [4]_. 
```

Comment:
@agramfort I just pushed another commit fixing that. Now Tipping and Fletcher are